src/main/java/dev/su5ed/sinytra/connector/locator/ConnectorLocator.java

```diff
@@ -176,6 +176,23 @@ private Stream<Path> scanClasspath() {
         }
     }
 
+    private Stream<Path> scanArguments() {
+        Stream<Path> pathStream = Arrays.stream(System.getProperty("connector.addMods", "").split(File.pathSeparator)).map(Path::of);
```

Comment:
```suggestion
        Stream<Path> pathStream = Arrays.stream(System.getProperty("connector.additionalModLocations", "").split(",")).filter(s -> !s.isBlank()).map(Path::of);
```
Path separators are annoying and differ from system to system.
`additionalModLocations` is also a better name as `addMods` isn't very descriptive.